src/sections/Community/Members-grid/membersGrid.style.js

```diff
@@ -1,8 +1,100 @@
 import styled from "styled-components";
+import icon5 from "../../../assets/images/layer5/5 icon/svg/light/5-light-no-trim.svg";
+import meshmateIcon from "../../../assets/images/meshmate/meshmate-icon.svg";
+
+const dot = (color = "#ccc") => ({
+  alignItems: "center",
+  display: "flex",
+
+  ":before": {
+    backgroundColor: color,
+    borderRadius: 10,
+    content: "\" \"",
+    display: "block",
+    marginRight: 8,
+    height: 10,
+    width: 10,
+    zindex: 1000,
+  },
+});
+
+const img = (icon) => ({
+  alignItems: "center",
+  display: "flex",
+  backgroundImage: `url(${icon})`,
+  backgroundRepeat: "no-repeat",
+  backgroundSize: 20,
+  backgroundPosition: "left center",
+
+  ":before": {
+    borderRadius: 10,
+    content: "\" \"",
+    display: "block",
+    marginRight: 12,
+    marginLeft: 5,
+    height: 10,
+    width: 10,
+    zindex: 1000,
+  },
+});
 
 export const selectStyles = {
-  menu: styles => ({ ...styles, zIndex: 999, })
+  menu: styles => ({ ...styles, zIndex: 999, }),
+  control: styles => ({ ...styles, backgroundColor: "white", zIndex: 900 }),
+  option: (styles, { data, isFocused, isSelected }) => ({
+    ...styles,
+
+    alignItems: "center",
```

Comment:
This is where I was sceptical about going against DRY. Any clever work-around for this?